src/i18n/DopplerIntlProvider.js

```diff
@@ -14,7 +14,11 @@ const messages = {
 addLocaleData([...en, ...es]);
 
 export default ({ locale, children }) => (
-  <IntlProvider locale={locale || 'en'} messages={flattenMessages(messages[locale || 'en'])}>
+  <IntlProvider
+    key={locale}
```

Comment:
```suggestion
    key={locale || 'en'}
```

Like in `locale` property?

Reply:
fixed